src/market/proposal.ts

```diff
@@ -13,7 +13,7 @@ export type ProposalProperties = Record<string, string | number | string[] | num
   "golem.activity.caps.transfer.protocol": string[];
   "golem.com.payment.debit-notes.accept-timeout?": number;
   "golem.com.payment.platform.erc20-polygon-glm.address"?: string;
-  "golem.com.payment.platform.erc20-goerli-tglm.address"?: string;
+  "golem.com.payment.platform.erc20-holesky-tglm.address"?: string;
```

Comment:
Has goerli been replaced by holesky? or maybe holesky is another tetsnet that can be used? if both testnets are available, there should be an additional property instead of replacement..

Reply:
https://www.alchemy.com/blog/goerli-faucet-deprecation

Goerli is going to be shut down this January, I'd say that's a proper replacement :)